nbt-extra-kotlin/src/main/kotlin/net/kyori/adventure/nbt/extra/kotlin/ListBinaryTagConverters.kt

```diff
@@ -0,0 +1,47 @@
+package net.kyori.adventure.nbt.extra.kotlin
+
+import net.kyori.adventure.nbt.BinaryTag
+import net.kyori.adventure.nbt.BinaryTagTypes
+import net.kyori.adventure.nbt.ByteArrayBinaryTag
+import net.kyori.adventure.nbt.ByteBinaryTag
+import net.kyori.adventure.nbt.DoubleBinaryTag
+import net.kyori.adventure.nbt.FloatBinaryTag
+import net.kyori.adventure.nbt.IntArrayBinaryTag
+import net.kyori.adventure.nbt.IntBinaryTag
+import net.kyori.adventure.nbt.ListBinaryTag
+import net.kyori.adventure.nbt.LongArrayBinaryTag
+import net.kyori.adventure.nbt.LongBinaryTag
+import net.kyori.adventure.nbt.ShortBinaryTag
+import net.kyori.adventure.nbt.StringBinaryTag
+
+public fun <T : BinaryTag> List<T>.toBinaryTag(): ListBinaryTag = ListBinaryTag.from(this)
+
+@JvmName("toBinaryTagByte")
```

Comment:
```suggestion
@JvmName("toByteBinaryTag")
```

Reply:
I can change these, though I think that it doesn't really matter what they're called tbh, since it's only designed to make it readable in Java, which we couldn't care less about. I am tempted to go through and do what I did with all of my extensions in another project, which is make them `@JvmSynthetic` so they don't show up in Java at all. I will go ahead and rename these, but I'll also add the synthetics, since I think it's a good idea this stuff is hidden from Java anyway, so that people don't go and accidentally use it where they shouldn't.